### PR TITLE
chore: update example tests

### PR DESCRIPTION
**Pull request by jawnsy**
Update tests for example values files to coalesce values using
the built-in Helm handling, rather than our custom rendering.
We should only use the struct-based rendering for unit tests.

### CODE REVIEW

**Review by deansheather — APPROVED** [review]

**Comment by jawnsy** [comment]
@jsjoeio This should unblock #166 once merged

**Review by jsjoeio — APPROVED** [review]
Hooray! This looks great 🚀

**Pull request merged**
